packages/slidev/node/vite/hmrPatch.ts

```diff
@@ -10,9 +10,7 @@ export function createHmrPatchPlugin(): Plugin {
     transform(code, id) {
       if (!id.match(regexSlideSourceId))
         return
-      const replaced = code.replace('if (_rerender_only)', 'if (false)')
-      if (replaced !== code)
-        return replaced
+      return code.replace('if (_rerender_only)', 'if (false)')
```

Comment:
Why this get removed? In Vite plugin returning bullish value will bailout the transform process and will be more performant

Reply:
Because the module has been tested to be a slide module, it is almost impossible to not perform the replacement